src/components/PositionListItem/index.tsx

```diff
@@ -221,9 +222,6 @@ export default function PositionListItem({
       {priceLower && priceUpper ? (
         <RangeLineItem>
           <RangeText>
-            <ExtentsText>
```

Comment:
@callil want to verify this looks like a bad merge to me, you didn't remove ExtentsText right?

Reply:
Bad merge

Reply:
reminder to re-add `ExtentsText`